src/test/java/it/com/isroot/stash/plugin/SettingsTest.java

```diff
@@ -0,0 +1,150 @@
+package it.com.isroot.stash.plugin;
+
+import com.atlassian.pageobjects.TestedProductFactory;
+import com.atlassian.stash.async.AsyncTestUtils;
+import com.atlassian.stash.async.WaitCondition;
+import com.atlassian.webdriver.pageobjects.WebDriverTester;
+import com.atlassian.webdriver.stash.StashTestedProduct;
+import com.atlassian.webdriver.stash.page.StashLoginPage;
+import org.hamcrest.Description;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Integration test to verify that both the YACC global and repo settings page are in sync and both working.
+ *
+ * @author Sean Ford
+ * @since 2015-08-30
+ */
+public class SettingsTest {
```

Comment:
@jimbethancourt , you might be interested in this. To help verify that the new global settings page works, I created this integration test that uses Selenium to do a few spot checks on both settings pages

* verify default values
* spot check some field errors
* make sure all fields are persisted after save

The tests aren't as clean as I would like (yet!), but it should provide some test coverage to ensure changes on one settings page won't break the other.